android/java/org/chromium/chrome/browser/app/BraveActivity.java

```diff
@@ -562,15 +573,21 @@ public void finishNativeInitialization() {
             BraveRewardsHelper.setNextRewardsOnboardingModalDate(calender.getTimeInMillis());
         }
 
-        if (SharedPreferencesManager.getInstance().readInt(BravePreferenceKeys.BRAVE_APP_OPEN_COUNT)
+        /*if
```

Comment:
Are we going to use commented code going forward? If not I'd just remove it.

Reply:
removed it